Core/QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/controller/QueryController.java

```diff
@@ -153,12 +153,74 @@ private void setup() {
 			@RequestParam(name = "options", required = false) List<String> options,
 			@RequestParam(name = "services", required = false) Boolean showServices) {
 		return getQueryData(request, request.getQueryString(), request.getParameterMap(), attrs, limit, offset, qToken,
-				options, showServices, false);
+				options, showServices, false,null);
+	}
+	
+	@GetMapping(path = "/type")
```

Comment:
[Minor] It should be "types" instead of type.
ex: /ngsi-ld/v1/types

Reply:
Done